Mijke jetten patch 2 (#153)
elixir-europe-training/ELIXIR-TrP-FAIR-Converge: docs/units/FAIR-generics/topics/1-why-fair.md

      time: 20 min
      type: individual exercise
      level: beginner 
    
    - learning_outcome: 1
      activities: Have participants present examples of different stakeholders (e.g., researchers, funders, the public) and discuss how each benefits from FAIR principles
      time: 10 min
      type: group discussion
      level: beginner 

    - learning_outcome: 2
      activities: Have participants provide a list of FAIR data requirements from journals and funding bodies and review them together
      time: 10 min
      type: group discussion
      level: beginner

    - learning_outcome: 2
      activities: In pairs, participants look up sample guidelines from a journal or funder and list how they impact data management practices
      time: 30 min
      type: group exercise
      level: beginner

    - learning_outcome: 3
      activities: Have participants list common issues in research that arise from non-FAIR data practices, such as data loss or inaccessibility
      time: 10 min
      type: group discussion
      level: beginner

    - learning_outcome: 3

>       activities: Ask participants to analyze case studies of projects that failed due to non-FAIR practices and discuss the repercussions

Reviewer: ```suggestion
      activities: Ask participants to analyse case studies of projects that failed due to non-FAIR practices and discuss the repercussions
```

      time: 20 min
      type: group discussion
      level: intermediate

    - learning_outcome: 4
      activities: Introduce participants to key changes needed to adopt FAIR principles within a research team, using simple examples
      time: 10 min
      type: group discussion
      level: beginner

    - learning_outcome: 4

>       activities: In groups, participants analyze a project scenario and identify specific changes needed to meet FAIR principles, sharing their findings

Reviewer: ```suggestion
      activities: In groups, participants analyse a project scenario and identify specific changes needed to meet FAIR principles, sharing their findings
```

      time: 20 min
      type: group discussion
      level: intermediate

    - learning_outcome: 4
      activities: Divide participants into groups to identify and share how each stakeholder’s needs align with FAIR principles in a research project
      time: 20 min
      type: group discussion
      level: intermediate
      
    - learning_outcome: 4

>       activities: Organize a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement FAIR principles

Reviewer: ```suggestion
      activities: Organise a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement the FAIR principles
```

>       activities: Organize a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement FAIR principles

Reviewer: ```suggestion
      activities: Organise a workshop where participants evaluate a real or hypothetical project’s current practices, then develop a detailed action plan to implement FAIR principles
```

  - title: Ten reasons to share your data
    author: Amanda S. Barnard
    url: https://www.natureindex.com/news-blog/ten-reasons-to-share-your-data

>   - title: FAIR Cookbook: What are the FAIR Principles?

Reviewer: I am not sure whether the ":" can cause issues. They did sometimes in the old version.

Author: Maybe @martijnkersloot knows (and can adjust if necessary)

Reviewer: I verified they did indeed cause errors. I removed them (see commits below).

  - title: The FAIR principles of data management
    url: https://genestack.com/resources/library/the-fair-principles-of-data-management/
    additionalInformation: section about stakeholders

>   - title: Implementing FAIR in data sharing: who are the stakeholders and what are their responsibilities?

Reviewer: As above, check if the ":" causes issues here.

Author: Maybe @martijnkersloot knows (and can adjust if necessary)
